contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -308,55 +308,36 @@ <h4 class="talent_name"></h4>
 
     <script>
         var all_capability = [];
-        var all_talents = [];
-        var get_talent_url = "http://localhost:5000/contributions/talents";
-        var get_cap_url = "http://localhost:5000/contributions/capabilities";
-
-        $(document).ready(function(){
-            var settings = {
-                      'cache': false,
-                      'dataType': "jsonp",
-                      "async": true,
-                      "crossDomain": true,
-                      "url": get_cap_url,
-                      "method": "GET",
-                      "headers": {
-                          "accept": "application/json",
-                          "Access-Control-Allow-Origin":"*"
-                      }
-                  }
-
-              $.ajax(settings).done(function (response) {
-                  console.log(response);
-                  all_capability = response;
-              });
-
-            var settings = {
-                  'cache': false,
-                  'dataType': "jsonp",
-                  "async": true,
-                  "crossDomain": true,
-                  "url": get_talent_url,
-                  "method": "GET",
-                  "headers": {
-                      "accept": "application/json",
-                      "Access-Control-Allow-Origin":"*"
-                  }
-              }
-
-              $.ajax(settings).done(function (response) {
-                  console.log(response);
-                  all_talents = response
-              });
-
-
-            append_capability(all_capability);
-            append_talent(all_talents);
-            filterSelection("all");
-            // append_capability(sample_contribution.capabilities);
-            // append_talent(sample_contribution.talents);
+        var all_talents = []
+        var get_talent_url = "http://"  + "localhost:5000/contributions/talents";
```

Comment:
Does this work if we deploy it to the actual server? Should this be relative path instead of static?